metatron/src/main/java/uk/gov/ida/eidas/metatron/resources/MetatronResource.java

```diff
@@ -43,6 +58,28 @@ public String truststore() throws KeyStoreException {
         });
         return trustedCerts.toString();
     }
+
+    @GET
+    @Path("/metadata/{entityId}")
+    public String metadata(@PathParam("entityId") String entityId) throws ResolverException {
```

Comment:
This and the `last-refresh` route are for testing too?

Reply:
Yeah, this and the are just a temporary things so we have something to poke whilst we're bringing it into something useful.  I guess if we use Prometheus, there might only be one endpoint that returns the MetatronResponse [sic].  At the moment, the entityId is the easiest thing to resolve without lots of extra code.